core/lib/spree/core/unreturned_item_charger.rb

```diff
@@ -53,12 +53,12 @@ def charge_for_items
 
     end
 
-    private
-
     def new_order
       @new_order ||= Spree::Order.create!(exchange_order_attributes)
```

Comment:
since we're making this public it seems like we ought to make it a bit safer to call.  i.e. calling `.new_order` probably shouldn't mutate anything right?  how about setting it explicitly inside `#charge_for_items` and just using an attr_reader to expose it?